packages/entities/entities-upstreams-targets/src/composables/useHelpers.ts

```diff
@@ -70,31 +70,39 @@ export default function useHelpers() {
           ? Object.entries(response.healthchecks.active.headers)
             .map(([key, val]) => ({ key, values: val?.join(', ') }))
           : [{ key: '', values: '' }],
-        interval: response?.healthchecks.active?.healthy?.interval?.toString() || '0',
-        successes: response?.healthchecks.active?.healthy?.successes?.toString() || '5',
-        httpStatuses: response?.healthchecks.active?.healthy?.http_statuses
-          ? numberToStringArray(response.healthchecks.active.healthy.http_statuses)
-          : ActiveHealthyHttpStatuses,
-        unhealthyInterval: response?.healthchecks.active?.unhealthy?.interval?.toString() || '0',
-        httpFailures: response?.healthchecks.active?.unhealthy?.http_failures?.toString() || '5',
-        tcpFailures: response?.healthchecks.active?.unhealthy?.tcp_failures?.toString() || '5',
-        unhealthyHttpStatuses: response?.healthchecks.active?.unhealthy?.http_statuses
-          ? numberToStringArray(response.healthchecks.active.unhealthy.http_statuses)
-          : ActiveUnhealthyHttpStatuses,
-        unhealthyTimeouts: response?.healthchecks.active?.unhealthy?.timeouts?.toString() || '0',
```

Comment:
I don't know why here didn't populate the default value from the schema.
Maybe there is something wrong. Let me check it again.
@sumimakito